docs/guides/audio-tracks.md

```diff
@@ -1,69 +1,80 @@
 # Audio Tracks
+Audio tracks are a feature of HTML5 video for providing alternate audio track selections to the user, so that a track other than the main track can be played. Video.js offers a cross-browser implementation of audio tracks.
 
-Audio Tracks are a function of HTML5 video for providing alternative audio track selections to the user, so that a track other than the main track can be played. Video.js makes audio tracks work across all browsers. There are currently five types of tracks:
+## Caveats
+- It is not possible to add audio tracks through HTML. They must be added [programmatically](#api).
+- Literal switching of audio tracks for playback is not handled by Video.js and must be handled by something else. Video.js only stores the track representation.
 
-- **Alternative**: alternative audio for the main video track
-- **Descriptions**: descriptions of what is happening in the video track
-- **Main**: the main audio track for this video
-- **Translation**: a translation of the main audio track
-- **Commentary**: commentary on the video, usually the director of the content talking about design choices
-
-## Missing Funtionality
-- It is currently impossible to add AudioTracks in a non-programtic way
-- Literal switching of AudioTracks for playback is not handled by video.js and must be handled by something else. video.js only stores the track representation
-
-## Adding to Video.js
-
-> Right now adding audio tracks in the HTML is unsupported. Audio Tracks must be added programatically.
-
-You must add audio tracks [programatically](#api) for the time being.
-
-## Attributes
-Audio Track propertites and settings
-
-### kind
-One of the five track types listed above. Kind defaults to empty string if no kind is included, or an invalid kind is used.
-
-### label
-The label for the track that will be show to the user, for example in a menu that list the different languages available for audio tracks.
+## Working with Audio Tracks
+### Add an Audio Track to the Player
+```js
+// Create a player.
+var player = videojs('my-player');
+
+// Create a track object.
+var track = new videojs.AudioTrack({
+  kind: 'translation',
+  label: 'Spanish',
+  language: 'es'
+});
 
-### language
-The two-letter code (valid BCP 47 language tag) for the language of the audio track, for example "en" for English. A list of language codes is [available here](languages.md#language-codes).
+// Add the track to the player's audio track list.
+player.audioTracks().addTrack(track);
```

Comment:
It would be a good idea to add examples for the rest of the audioTrack and audioTrackList functions well. IE: `removeTrack`, `addTrack`, `getTrackById`